From 20b7f912442f80995bbf2e28691188beaeaf8897 Mon Sep 17 00:00:00 2001
From: Ben Boeckel <MathStuf@gmail.com>
Date: Mon, 2 Nov 2009 10:58:50 -0500
Subject: [PATCH] Don't ship forked simplejson

---
 MANIFEST.in |    3 ---
 1 files changed, 0 insertions(+), 3 deletions(-)

diff --git a/MANIFEST.in b/MANIFEST.in
index 1ec617c..8a36047 100644
--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -7,9 +7,6 @@ include darwin/*
 include jsonpickle/LICENSE
 include jsonpickle/*.py
 include jsonpickle/tests/*.py
-include simplejson/LICENSE
-include simplejson/*.py
-include simplejson/tests/*.py
 include test/*.py
 include test/Makefile
 include README
-- 
1.6.5.1

