diff -ur cola-1.4.3/setup.py cola-1.4.3-translations/setup.py
--- cola-1.4.3/setup.py	2010-12-28 04:03:25.000000000 +0100
+++ cola-1.4.3-translations/setup.py	2011-01-03 02:21:01.000000000 +0100
@@ -95,12 +95,13 @@
             _app_path('share/git-cola/styles/images', '*.png'),
             _app_path('share/applications', '*.desktop'),
             _app_path('share/doc/git-cola', '*.txt'),
-            _app_path('share/locale', '*/LC_MESSAGES/git-cola.mo'),
             _package('cola'),
             _package('cola.models'),
             _package('cola.controllers'),
             _package('cola.views')]
 
+    data.extend([_app_path(localedir, 'git-cola.mo') for localedir in glob('share/locale/*/LC_MESSAGES')])
+
     if not standalone:
         data.extend([_thirdparty_package('jsonpickle'),
                      _thirdparty_package('simplejson')])
